service/routes.py

```diff
@@ -6,7 +6,7 @@
 
 from flask import Flask, jsonify, request, url_for, make_response, abort
 from service.common import status  # HTTP Status Codes
-from service.models import Inventory
+from service.models import Inventory, Condition
```

Comment:
I meant to make this change in the next sprint

The mainline version of routes.py already has import Condition, maybe you can get rid of that line when you merge?

Reply:
I have fixed that.